plugins/go-build/bin/goenv-install

```diff
@@ -123,9 +127,15 @@ DEFINITION="${ARGUMENTS[0]}"
 
 # NOTE: Try to capture semantic versions such as `1.11` which don't have a patch version.
 # A patch version of 0 will be added, e.g they'll be changed to `1.11.0`.
-if grep -q -E "^[0-9]+\.[0-9]+(\s*)$" <<< ${DEFINITION}; then
-  echo "Adding patch version 0 to ${DEFINITION}"
-  DEFINITION="${DEFINITION}.0"
+if [[ ${DEFINITION} == "latest" ]]; then
+  LATEST=$(latest_version "[0-9]\\.[0-9]+")
+  echo "Installing latest version ${LATEST}..."
+  DEFINITION=$LATEST
+elif grep -q -E "^[0-9]+\.[0-9]+(\s*)$" <<<${DEFINITION}; then
+  REGEX=$(echo $DEFINITION | sed s/\\./\\\\./)
+  LATEST_PATCH=$(latest_version $REGEX)
+  echo "Using latest patch version $LATEST_PATCH"
+  DEFINITION="$LATEST_PATCH"
```

Comment:
I think the `NOTE` above needs to be updated to reflect the more complex scenarios covered here.

Reply:
Not sure I understand the ask on this one